### PR TITLE
Fix Sales Order

### PR DESCRIPTION
**Pull request by x4d3**
map customer and make better object_name

### CODE REVIEW

File: app/models/entities/sales_order.rb [hunk 1]
```diff
@@ -13,14 +13,21 @@ def self.mapper_class
...
+    entity['title']
   end
 
   def self.object_name_from_external_entity_hash(entity)
-    entity['title']
+    entity['name']
   end
 
   def map_to_connec(entity, organization)
+    if entity['customer']
```
Comment (Berardpi) [inline app/models/entities/sales_order.rb]: could avoid that with a 
```ruby
def self.references
  [{reference_class: Entities::Person, connec_field: 'person_id', external_field: 'customer/id'}]
end
```

File: app/models/entities/sales_order.rb [hunk 2]
```diff
@@ -33,6 +40,12 @@ def map_to_connec(entity, organization)
   end
 
   def map_to_external(entity, organization)
+    person_id = entity['person_id']
```
Comment (Berardpi) [inline app/models/entities/sales_order.rb]: references should take care of that too.

idmaps above and below should use helper:
```ruby
idmap = Entities::Item.find_idmap({organization_id: organization.id, external_id: id})
idmap = Entities::Item.find_idmap({organization_id: organization.id, connec_id: id})
```

File: app/models/entities/invoice.rb [hunk 3]
```diff
@@ -13,11 +13,11 @@ def self.mapper_class
   end
 
   def self.object_name_from_connec_entity_hash(entity)
-    entity['description']
+    entity['code']
   end
 
   def self.object_name_from_external_entity_hash(entity)
-    entity['title']
+    nil
```
Comment (Berardpi) [inline app/models/entities/invoice.rb]: `nil`?

Comment (x4d3) [inline app/models/entities/invoice.rb]: I don't know what else should I return.

**Pull request merged**